styles.css

```diff
@@ -71,25 +72,86 @@ logo {
 
 .centerbtn {
     display: flex;
-    justify-content: center; 
-    /* background-color: var(--primary-color);
-    border: none;
-    color: var(--tertiary-color);
-    width: 150px;
-    height: 40px;
-    font-size: 16px;
-    border-radius: 5px;
-    padding: 10px;  */
+    justify-content: center;
 }
 
 .footer {
     display: flex;
     align-items: center;
     padding: 10px;
-    /* background: var(--primary-color); */
-    text-decoration: none;
     color: var(--secondary-color);
     position: absolute;
     bottom: 0; 
 }
 
+.wrapper {
+    max-width: ;
+    background: var(--secondary-color);
+    box-shadow: 0 4px 4px 0 rgba(0, 0, 0, 0.05);
+    order-radius: 5px;
+    padding: 1rem;
+    margin: 1rem auto;
```

Comment:
Good to see you have used rem.
 I suggest you to use rem than px. check this throughout code.

Reply:
made the changes.